CHANGELOG.md

```diff
@@ -91,6 +91,7 @@ New deprecation(s):
 - **GitHub Runner Scaler**: Fix explicit repo check 404 to skip not crash ([#4790](https://github.com/kedacore/keda/issues/4790))
 - **GitHub Runner Scaler**: Fix rate checking on GHEC when HTTP 200 ([#4786](https://github.com/kedacore/keda/issues/4786))
 - **Pulsar Scaler**: Fix `msgBacklogThreshold` field being named wrongly as `msgBacklog` ([#4681](https://github.com/kedacore/keda/issues/4681))
+- **General**: Add one deployment arg in keda-metrics-apiserver for only printing errors, but errors should go to stderr ([#4049](https://github.com/kedacore/keda/issues/4049))
```

Comment:
Could you please move this up to Unreleased section, to ### Other subsection? Thanks!

Reply:
sure @zroubalik , actually there are some problem with my side and branches during update, so I am closing this PR and will again raised regarding same things

Reply:
I am sorry for  this inconvenience .